keyboards/rainkeebs/trailmix/rules.mk

```diff
@@ -0,0 +1,4 @@
+# Build Options
+#   change yes to no to disable
+#
+ENCODER_MAP_ENABLE = yes
```

Comment:
@waffle87 as I am using [encoder mapping](https://github.com/qmk/qmk_firmware/blob/master/docs/feature_encoders.md#encoder-map-idencoder-map), as suggested in the previous [comment](https://github.com/qmk/qmk_firmware/pull/20189#discussion_r1142504827), on the keymap level, I need to have leave it to compile the encoders behaviour. Should I use [callbacks approach](https://github.com/qmk/qmk_firmware/blob/master/docs/feature_encoders.md#callbacks) and remove it from there instead?
Just to understand which is the best approach

Reply:
Best approach is to have `ENCODER_MAP_ENABLE` & its relevant code at the keymap level, and `ENCODER_ENABLE` & the `encoder_update_kb` function at the keyboard level.

Reply:
Perfect!
I will implement both.
Thanks so much for your help